modules/flowable-process-validation/src/main/java/org/flowable/validation/validator/impl/ExternalInvocationTaskValidator.java

```diff
@@ -42,13 +46,16 @@ protected void validateFieldDeclarationsForEmail(org.flowable.bpmn.model.Process
             if (fieldExtension.getFieldName().equals("textVar")) {
                 textOrHtmlDefined = true;
             }
+            if (fieldExtension.getFieldName().equals("templateid")) {
```

Comment:
I'm not a fan of the fact that these variables are hard coded + the for loop could be improved (but minimal changes)

Reply:
Yes I agree